standardGameBoard.js

```diff
@@ -0,0 +1,73 @@
+import { Nation } from "./constants";
+
+export default {
```

Comment:
What if the default export of this module were an instance of `GameBoard` instead of the arguments required to construct it?